front/package-lock.json

```diff
@@ -10736,9 +10736,9 @@
       "license": "Apache-2.0"
     },
     "node_modules/@dust-tt/sparkle": {
-      "version": "0.2.193",
-      "resolved": "https://registry.npmjs.org/@dust-tt/sparkle/-/sparkle-0.2.193.tgz",
-      "integrity": "sha512-ehUgBTsBv6kqZD/eWxeZ7CD5fMRBBSrkddCvs6YMI+QxD1yhpLut4mc9aJ0XMreVx6FwHnQCoqkWMisgZtT4fQ==",
+      "version": "0.2.195",
```

Comment:
Weird that you bump only the package-lock.json, expected?

Reply:
Yeah, package was supposedly bumped there https://github.com/dust-tt/dust/pull/6577/files#diff-1335400798ac3f1df5513672e631b9269f8febecf18045d2dd4da05afcbe81c9 but forgot to bump front 🤷

Reply:
Yeah but then it should bump the pacakge.json as well, no?